lib/galaxy/webapps/base/controller.py

```diff
@@ -1577,3 +1577,15 @@ def sort_by_attr(seq, attr):
     intermed = [(getattr(v, attr), i, v) for i, v in enumerate(seq)]
     intermed.sort()
     return [_[-1] for _ in intermed]
+
+
+__all__ = (
+    "HTTPBadRequest",
```

Comment:
Re-exported, imported from here in `lib/galaxy/webapps/galaxy/api/library_contents.py` , `lib/tool_shed/webapp/api/repositories.py` , `lib/tool_shed/webapp/api/repository_revisions.py`